Feature/updates (#59)
infinum/eightshift-docs: website/docs/basics/blocks-component-in-block.md

"content": "test",
```

> > Keep in mind that we handle the attributes data necessary to have multiple components in one block but you must provide the correct props inside you files to mock the custom attribute names. Best way to check this block and see what we have done here: [Card Block](https://github.com/infinum/eightshift-frontend-libs/tree/develop/blocks/init/src/Blocks/custom/card)

Reviewer: ```suggestion
> Keep in mind that we handle the attributes data necessary to have multiple components in one block but you must provide the correct props inside your files to mock the custom attribute names. Best way to check this block and see what we have done here: [Card Block](https://github.com/infinum/eightshift-frontend-libs/tree/develop/blocks/init/src/Blocks/custom/card)
```

> > Keep in mind that we handle the attributes data necessary to have multiple components in one block but you must provide the correct props inside you files to mock the custom attribute names. Best way to check this block and see what we have done here: [Card Block](https://github.com/infinum/eightshift-frontend-libs/tree/develop/blocks/init/src/Blocks/custom/card)

Reviewer 2: ```suggestion
> Keep in mind that we handle the attributes data necessary to have multiple components in one block but you must provide the correct props inside your files to mock the custom attribute names. The best way to see how this is done is to check this block for example: [Card Block](https://github.com/infinum/eightshift-frontend-libs/tree/develop/blocks/init/src/Blocks/custom/card)
```
